shell/components/GlobalRoleBindings.vue

```diff
@@ -364,6 +370,11 @@ export default {
 </style>
 <style lang='scss' scoped>
   $detailSize: 11px;
+
+  .deprecation-notice {
+    margin: 8px 0 8px 20px;
+    color: var(--error);
```

Comment:
I think we should remove the red - it stands out too much - just having the text is sufficient, I think.